src/google_sheets_package/table_fields_serializer.py

```diff
@@ -0,0 +1,25 @@
+class TableFieldsSerializer:
```

Comment:
Такое уже было, надо менять на что-то такое

```suggestion
class GSheetFields:
```

```suggestion
class TableFields:
```